assets/less/site/elements/button.overrides

```diff
@@ -101,3 +101,27 @@ button {
     //   0% 0.63rem
     // );
   }
+
+  .img-button {
+    all: unset;
+    cursor: pointer;
+    display: inline-flex;
+    align-items: center;
+    justify-content: center;
+  }
+
+  .close-icon {
+    width: 1.5rem;
+    height: 2rem;
+  }
+
+  .slide-icon {
+    width: 1rem;
+    height: 1.05rem;
+  }
+
+  img-button-icon {
```

Comment:
This should be a class, `.` is missing. Can you use nesting under `.img-button`?